src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -226,11 +226,10 @@ void Run (DirectoryAssemblyResolver res)
 			foreach (var kvp in managedConflicts) {
 				Log.LogCodedWarning (
 					"XA4214",
-					"The managed type `{0}` exists in multiple assemblies: {1}. " +
-					"Please refactor the managed type names in these assemblies so that they are not identical.",
+					Properties.Resources.XA4214,
 					kvp.Key,
 					string.Join (", ", kvp.Value));
```

Comment:
Since using the `XA4214` resource made it a lot shorter, should this `LogCodedWarning` just be on one line?

Otherwise looks good to me. 👍